sdks/java/io/hadoop-common/pom.xml

```diff
@@ -32,6 +32,17 @@
 
   <dependencies>
     <dependency>
+      <groupId>org.apache.beam</groupId>
```

Comment:
Yes, that's the reason, do you think this is ok (considering that the other libraries depending on this one probably will have the same dependency) ?